joomla/joomla-cms
# [com_finder] maps view: Use title in language subbranch

#### Summary of Changes

This make a small adjustement to the language subbranches in smart search content maps.
##### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/15081225/cfde6870-13b9-11e6-9b4e-71d5e4565c4a.png)
#### Testing Instructions
1. Apply patch
2. Check the `Language` subbranches in Components -> Smart Search -> Content Maps
   
   Before PR: `en-GB` and so on.
   
   After PR: the content language title.
   The same you have in Extensions -> Languages -> Content Languages
